packages/paste-website/src/pages/components/help-text/index.mdx

```diff
@@ -0,0 +1,342 @@
+    </Box>
+  </Do>
+  <Dont
+    title="Don't"
+    body="Don't have a character limit if you can't explain to the user why their text entry is restricted."
+    center
+  >
+    <Box width="100%" padding="space60">
+      <Label htmlFor="limited_dont">Abbreviated country</Label>
+      <Input id="limited_dont" name="limited_dont" type="text" onChange="" />
+    </Box>
+  </Dont>
+</DoDont>
+<DoDont>
+  <Do
+    title="Do"
+    body="Keep help text and error text concise and simple. If you need to use more than 2 sentences to explain a field, link out to supporting docs or trigger a popover instead."
+    center
+  >
+    <Box width="100%" padding="space60">
+      <Label htmlFor="help_do">Phone number</Label>
+      <Input aria-describedby="help_do_help_text" id="help_do" name="help_do" type="tel" onChange="" />
+      <HelpText id="help_do_help_text">Use the following format: (###) ###-####</HelpText>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don't use more than 2 sentences in help text or error text." center>
+    <Box width="100%" padding="space60">
+      <Label htmlFor="help_dont">Email address</Label>
+      <Input aria-describedby="help_dont_help_text" id="help_dont" name="help_dont" type="email" onChange="" />
+      <HelpText id="help_dont_help_text">
+        Go to your settings. Then click on email addresses. After doing that, copy and paste your email address in this
+        field.
+      </HelpText>
+    </Box>
+  </Dont>
+</DoDont>
+
+## Anatomy
+
+<Box marginBottom="space60">
+  <Table tableLayout="fixed">
+    <THead>
+      <Tr>
+        <Th>Property</Th>
+        <Th>Default token</Th>
+        <Th>Modifiable?</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Td>Help text</Td>
+        <Td>$color-text-weak, $font-size-30</Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Inline error</Td>
+        <Td>
+          <UnorderedList marginBottom="space0">
+            <ListItem>Text: $color-text-error, $font-size-30</ListItem>
+            <ListItem>Icon: IconError, $color-text-error, $icon-size-20</ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Spacing</Td>
+        <Td>
+          <UnorderedList marginBottom="space0">
+            <ListItem>
+              Help text:
+              <UnorderedList marginBottom="space0">
+                <ListItem>Top: $space-30</ListItem>
+              </UnorderedList>
+            </ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</Box>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/help-text - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {Input, Label, HelpText} from '@twilio-paste/core';
+
+const Component = () => (
+  <>
+    <HelpText id="foo_text">Please enter some text</HelpText>
```

Comment:
Do you think we should show a more complete example with the Input and Label here?

Reply:
Yep, good catch! I'll add it now.